src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/__tests__/ChartBuilder.test.tsx

```diff
@@ -197,7 +197,7 @@ describe('ChartBuilder', () => {
       expect(updatedTableRows[2]).toHaveTextContent(
         'Number of authorised absence sessions (Ethnicity Major Chinese, State-funded secondary, Barnet, 2014/15)',
       );
-    });
+    }, 20000);
```

Comment:
Do we need this still with the new runners?

Reply:
Yep seems so. It happens consistently locally